recipes/libelfin/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "0.3":
+    url: "https://github.com/aclements/libelfin/archive/v0.3.tar.gz"
+    sha256: "c338942b967582922b3514b54b93175ca9051a9668db92dd8ef619824d443ac7"
+patches:
+  "0.3":
+    - patch_file: "patches/9d0db16d0a0b3c4f8aaa60a3e4dab295df34b6b2.patch"
```

Comment:
Can you provide an explicit file name ?

Reply:
The patch directly related to commit `9d0db16d0a0b3c4f8aaa60a3e4dab295df34b6b2` from the original repository. The commit goes on top of the release and there was no release available.
Will something like `commit-9d0db16d0a0b3c4f8aaa60a3e4dab295df34b6b2.patch` work better?

Reply:
Yes, prefixing it with `commit-` is much better.

Reply:
https://github.com/conan-io/conan-center-index/pull/3283/commits/1bce461afd70453f4a3ac2ece2c44bc8d7eef104